demos/Makefile

```diff
@@ -1,19 +1,14 @@
-cases := base_case 2d_compressible_ALA 2d_compressible_TALA viscoplastic_case 2d_cylindrical analytical_comparisons Drucker-Prager_rheology 3d_spherical 3d_cartesian adjoint optimisation_checkpointing scalar_advection_diffusion multi_material
-long_cases := analytical_comparisons parallel_scaling
+cases := base_case 2d_compressible_ALA 2d_compressible_TALA viscoplastic_case 2d_cylindrical Drucker-Prager_rheology 3d_spherical 3d_cartesian adjoint
```

Comment:
Is this a list of all demos that will be carried forward? If so, should multi-material examples be here too?

Reply:
This is all the demos that double as regression tests. The multi-material demos themselves don't currently (in master) get run, except for when building the notebooks, but I guess they actually should to make sure they work!